src/java/fr/paris/lutece/plugins/mydashboard/modules/identity/resources/identity_messages_en.properties

```diff
@@ -193,4 +193,9 @@ view_complete_identity.lastName.label=Lastname
 view_complete_identity.birthdate.label=Date of birth
 view_complete_identity.continue=Continue
 view_complete_identity.error.birthplace=City of birth is required
-view_complete_identity.error.birthcountry=Country of birth is required
\ No newline at end of file
+view_complete_identity.error.birthcountry=Country of birth is required
+
+
+dashboard.indentityInfos.description=Dashboard identity - Informations d'identit\u00e9
```

Comment:
replace the labels in French with labels in English in identity_messages_en.properties